src/StreetNameRegistry.Api.Oslo/StreetName/Detail/StreetNameOsloResponse.cs

```diff
@@ -68,6 +68,9 @@ public class StreetNameOsloResponse
         [JsonProperty(Required = Required.DisallowNull)]
         public StraatnaamStatus StraatnaamStatus { get; set; }
 
+        [JsonIgnore]
+        internal string? LastEventHash { get; set; }
```

Comment:
Why is this added if it has the JsonIgnore attribute?

Reply:
The controller needs this to add to the etag header.